packages/admin-ui/src/index.js

```diff
@@ -11,5 +12,10 @@ if (process.env.NODE_ENV !== 'production' && process.env.REACT_APP_AXE) {
   axe(React, ReactDOM);
 }
 
-ReactDOM.render(<App />, document.getElementById('root'));
+api('admin_ui_routes').then(({ routes }) => {
```

Comment:
do you think its the best idea to do this that early in process? Can't we start to bootup the application  already and then  do the http request? I'm just curious what your thoughts are.

Reply:
I was trying to think about where to load these routes. If they are loaded as early as possible, there shouldn't be any delay (fetching additional available routes), if a user tries to navigate to the page that is part of a module provided route.

Reply:
I think it's hard to make any assumption at this point what problems we might run into with this. Maybe we should just go with the current solution and optimize it later based on feedback?

Reply:
👍